packs/equipment/iron-cube.json

```diff
@@ -11,7 +11,7 @@
         "containerId": null,
         "damage": null,
         "description": {
-            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">2</span> (concentrate)</p>\n<hr />\n<p>This cube of blackened iron is affixed to a weapon with an iron chain. When you activate the cube, make a melee Strike. If it hits and deals damage, you can attempt an Athletics check to @UUID[Compendium.pf2e.actionspf2e.Item.Trip] the creature you hit. If this knocks the target @UUID[Compendium.pf2e.conditionitems.Item.Prone], the target takes @Damage[2d6[bludgeoning]] damage from the force of the impact. If you're wielding a two-handed melee weapon, you can ignore Trip's requirement that you have a free hand. Both attacks count toward your multiple attack penalty, but the penalty doesn't increase until after you've made both of them.</p>\n<p>If you have the Knockdown feat, ignore the normal size restrictions for the trip and increase the damage dice if you knock the target prone to the damage die of your weapon if it's higher than d6.</p>"
+            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">2</span> (concentrate)</p><hr /><p>This cube of blackened iron is affixed to a weapon with an iron chain. When you activate the cube, make a melee Strike. If it hits and deals damage, you can attempt an Athletics check to @UUID[Compendium.pf2e.actionspf2e.Item.Trip] the creature you hit. If this knocks the target @UUID[Compendium.pf2e.conditionitems.Item.Prone], the target takes @Damage[2d6[bludgeoning]] damage from the force of the impact. If you're wielding a two-handed melee weapon, you can ignore Trip's requirement that you have a free hand. Both attacks count toward your multiple attack penalty, but the penalty doesn't increase until after you've made both of them.</p>\n<p>If you have the @UUID[Compendium.pf2e.feats-srd.Item.Slam Down] feat, ignore the normal size restrictions for the trip and increase the damage dice if you knock the target prone to the damage die of your weapon if it's higher than d6.</p>"
```

Comment:
The Knockdown feat was renamed in the remaster to Slam Down. Should we override the label? I assume this also needs to be reported to Paizo for errata?

Reply:
We're fine using the auto label.